eventhubs/client/azure-eventhubs/src/main/java/com/azure/eventhubs/EventHubClientBuilder.java

```diff
@@ -59,11 +69,11 @@ public EventHubClientBuilder transportType(TransportType transport) {
     /**
```

Comment:
Missing the method that we talked about, in case the user chose to use a SAS token from the namespace rather than a specific Event Hub.
```
public EventHubClientBuilder eventHubPath(String eventHubPath) {
}
```